.github/tekton/README.md

```diff
@@ -0,0 +1,15 @@
+# KS Repo CI/CD
+
+Why dose `ks-releaser` project have a folder `.github/tekton`? Cuz we want to replace GitHub workflow with Tekton.
```

Comment:
`Cuz` ?

Reply:
![image](https://user-images.githubusercontent.com/16865714/146879101-8695aa21-4cdb-48ac-901c-591588c8f416.png)

Reply:
As you can see, this is an informal word. Please use a formal word instead.